src/dat/view.cljs

```diff
@@ -266,14 +266,14 @@
 (representation/register-representation
   ::pull-summary-string
   (fn [_ _ pull-data]
-    ;[:span
+    [:span
```

Comment:
Oh... yes, it should. As discussed in Gitter, let's try using `re-com/label` instead. Thinking more about this, the compatibility issues I was having may have actually been in something outside of datview.